bmrc/templates/streams/webfeed_block.html

```diff
@@ -1,4 +1,6 @@
-<div class="column">
-    <h2>{{ self.webfeed_title }}</h2>
-    {{ self.webfeed_code }}
-</div>
\ No newline at end of file
+<section class="webfeed-block">
+    <div class="container-sm">
+        <h2>{{ self.webfeed_title }}</h2>
+        {{ self.webfeed_code }}
+    </div>

```

Comment:
I think the same applies here, these should probably be wrapped in `if` statements since they're not required fields.